dev/create_certificates_linux.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+dotnet tool install -g dotnet-certificate-tool
+
+stty -echo
+printf "Identity Server Dev Password: "
+read identity_server_dev_pass
+stty echo
+printf "\n"
+
+openssl req -x509 -newkey rsa:4096 -sha256 -nodes -keyout identity_server_dev.key -out identity_server_dev.crt \
+    -subj "/CN=Bitwarden Identity Server Dev" -days 3650
+openssl pkcs12 -export -out identity_server_dev.pfx -inkey identity_server_dev.key -in identity_server_dev.crt \
+    -certfile identity_server_dev.crt -password pass:$identity_server_dev_pass
+
+identity="$(openssl x509 -in identity_server_dev.crt -outform der | shasum -a 1 | head -c 40 | tr a-z A-Z)"
+
+dotnet certificate-tool add --file ./identity_server_dev.pfx --password $identity_server_dev_pass
```

Comment:
did you confirm the behavior of this if no password is given?

Reply:
I have now and had to add an if else for no password but it works.